model/patch_lifecycle.go

```diff
@@ -264,14 +264,12 @@ func GetPatchedProject(ctx context.Context, settings *evergreen.Settings, p *pat
 			return nil, nil, errors.Wrap(err, "finding and translating project")
```

Comment:
GH doesn't let me comment on the line but the comment on L261 has to be updated since the parser project is no longer stored as a string in the patch. The project to remove the parser project limit changed the parser project string to instead just store the document in the DB at the same time as the patch.

Reply:
good catch, thanks!